engine/src/main/java/org/destinationsol/world/generators/SolarSystemGeneratorImpl.java

```diff
@@ -22,14 +22,34 @@
  *
  * This class also has access to the featureGenerators list from {@link SolarSystemGenerator}.
  * This allows it to choose which FeatureGenerators to use in populating the SolarSystem.
- * TODO: Define the behavior of default Planets in this class (As it is implemented in the game currently)
+ * TODO: Define the behavior of default SolarSystems in this class (As it is implemented in the game currently)
  */
 public class SolarSystemGeneratorImpl extends SolarSystemGenerator {
 
     @Override
     public void build() {
-        for (FeatureGenerator generator : featureGenerators) {
-            generator.build();
+        setPlanetCount(5);
+        setPossibleBeltCount(1);
+        setMazeCount(2);
+        setSunCount(1);
```

Comment:
These four lines seem to me like they should really be one method with four parameters, possibly in the abstract class's constructor